tests/cms/test_dummy.py

```diff
@@ -0,0 +1,8 @@
+import json
+
+from integreat_compass.cms.models.justfortesting import add
+
+def test_dummy():
+    with open("integreat_compass/cms/fixtures/test_data.json", encoding="utf-8") as f:
+        result = json.load(f)
+    assert result == []
```

Comment:
```suggestion

def test_dummy(load_test_data):
    pass

```
Maybe this makes more sense as dummy test? This also checks whether the test data has been loaded via the fixture.